lib/location/method_gnss.c

```diff
@@ -610,20 +610,37 @@ static void method_gnss_positioning_work_fn(struct k_work *work)
 	/* Configure GNSS to continuous tracking mode */
 	err = nrf_modem_gnss_fix_interval_set(1);
 
-	uint8_t use_case;
+	uint8_t use_case = NRF_MODEM_GNSS_USE_CASE_MULTIPLE_HOT_START;
 
-	switch (gnss_config.accuracy) {
-	case LOCATION_ACCURACY_LOW:
-		use_case = NRF_MODEM_GNSS_USE_CASE_MULTIPLE_HOT_START |
-			   NRF_MODEM_GNSS_USE_CASE_LOW_ACCURACY;
-		err |= nrf_modem_gnss_use_case_set(use_case);
-		break;
+	if (gnss_config.accuracy == LOCATION_ACCURACY_LOW) {
+		use_case |= NRF_MODEM_GNSS_USE_CASE_LOW_ACCURACY;
+	}
 
-	case LOCATION_ACCURACY_NORMAL:
-		use_case = NRF_MODEM_GNSS_USE_CASE_MULTIPLE_HOT_START;
-		err |= nrf_modem_gnss_use_case_set(use_case);
-		break;
+#ifdef CONFIG_NRF_CLOUD_AGPS_FILTERED
```

Comment:
could this chunk of code be moved into a common function?  it is used 3 times in this PR

Reply:
I thought about that, but it's not clear to me where that would live, since it's not nRF Cloud-specific.  Any suggestions?

Reply:
done